openhtf/io/output/mfg_inspector.py

```diff
@@ -171,14 +171,14 @@ def _MangleMeasurement(name, value, measurement, mangled_parameters,
   We generate these by doing some name mangling, using some sane limits for
   very large multidimensional measurements.
   """
-  for current_value in value[:MAX_PARAMS_PER_MEASUREMENT]:
+  for coord, val in value.value_dict.iteritems():
```

Comment:
probably still want to respect `MAX_PARAMS_PER_MEASUREMENT` here, with `.items()[:MAX_PARAMS_PER_MEASUREMENT]` instead.